### PR TITLE
minor fixes & documentation

### PR DESCRIPTION
**Pull request by iloveitaly**
- docs: misc spelling error fixes
- build: ignoring local config files

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pdm 2.10.4
+python 3.11.6
```
Comment (evanpurkhiser) [inline .tool-versions]: What does the `.tool-versions` file do?

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,6 @@
 .pdm.toml
 __pypackages__
+.pdm-python
+.envrc
+.env
+*.db
```
Comment (evanpurkhiser) [inline .gitignore]: Could you bring back the trailing new-line?

File: .tool-versions (hunk 1 above)
Comment (iloveitaly) [reply .tool-versions]: it's for https://asdf-vm.com/

File: .gitignore (hunk 2 above)
Comment (iloveitaly) [reply .gitignore]:
```suggestion
*.db

```

Comment (iloveitaly) [reply .gitignore]: done!

File: .tool-versions (hunk 1 above)
Comment (evanpurkhiser) [reply .tool-versions]: I'm going to go ahead and leave this out since I don't use it and I probably won't maintain it!

**Pull request merged**